src/osemgrep/cli_interactive/Pointed_zipper.mli

```diff
@@ -42,9 +42,12 @@ type 'a t
 *)
 val move_left : 'a t -> 'a t
```

Comment:
would be good at some point to give small explanations for each of those functions.